Doc/library/socket.rst

```diff
@@ -669,6 +669,30 @@ The following functions all create :ref:`socket objects <socket-objects>`.
    .. versionadded:: 3.3
 
 
+.. function:: socket.send_fds(sock, buffers, fds[, flags[, address]])
+
+   :func:`socket.send_fds` sends the list of file descriptors *fds*
+   over an :const:`AF_UNIX` socket, on systems which support the
+   :const:`SCM_RIGHTS` mechanism.
```

Comment:
What are buffers, flags and address? Maybe suggest to read sendmsg() for the documentation of these parameters?